unified-runtime/test/layers/tracing/CMakeLists.txt

```diff
@@ -65,15 +29,7 @@ function(add_tracing_test name)
         ${PROJECT_NAME}::headers
         ${PROJECT_NAME}::testing
         GTest::gtest_main)
-    add_test(NAME ${name}
-        COMMAND ${CMAKE_COMMAND}
-        -D MODE=stderr
-        -D TEST_FILE=$<TARGET_FILE:${TEST_TARGET_NAME}>
-        -D MATCH_FILE=${CMAKE_CURRENT_SOURCE_DIR}/${name}.out.match
-        -P ${PROJECT_SOURCE_DIR}/cmake/match.cmake
-        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
-    )
-    set_tracing_test_props(${name} test_collector)
 endfunction()
 
 add_tracing_test(codeloc codeloc.cpp)
```

Comment:
The executable will exist in `build/bin/tracing-test-codeloc`. Is it worth updating this to build into `build/unittests/bin` or similar?

Reply:
if all other test executables are there i would say yes

Reply:
All executables, both testing and non-testing, live in `build/bin`. I'm wondering if it's worth taking the move to lit as an excuse to move tests into a more specific directory just to keep the testing stuff separate.